tests/containers/helm_k3s.pm

```diff
@@ -0,0 +1,70 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test deploy a helm chart in a k3s
+# - install k3s, kubectl and hem
+# - test helm repo add, update, search and show all
+# - add bitnami repo
+# - test helm install with apache helm chart
+# - test helm list
+# - check the correct deployment of the helm chart
+# - cleanup system (helm and k3s)
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use utils qw(script_retry);
+use mmapi 'get_current_job_id';
+use containers::k8s;
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_serial_terminal;
+    $self->{rancher_name} = "rancher-" . get_current_job_id();
+    $self->{deployment_name} = "apache-" . get_current_job_id();
+    my $chart = "bitnami/apache";
+
+    install_k3s();
+    install_kubectl();
+    install_helm();
+
+    # Add repo, search and show values
+    assert_script_run("helm repo add bitnami https://charts.bitnami.com/bitnami");
+    assert_script_run("helm repo update");
+    assert_script_run("helm search repo apache");
+    assert_script_run("helm show all $chart");
+
+    # Install apache
+    assert_script_run("helm install $self->{deployment_name} $chart");
+    assert_script_run("helm list");
+
+    # Wait for deployment
+    assert_script_run("kubectl rollout status deployment/$self->{deployment_name}");
+    assert_script_run("kubectl describe deployment/$self->{deployment_name}");
+    assert_script_run("kubectl get pods | grep $self->{deployment_name}");
+    assert_script_run("kubectl describe services/$self->{deployment_name}");
+
+    # Test
+    enter_cmd("kubectl port-forward  services/$self->{deployment_name} 10002:80 &");
+    script_retry("curl http://localhost:10002", delay => 30, retry => 5);
```

Comment:
Isn't this similat to what you do in EKS Helm test?

Reply:
mmm are you thinking that we could work on some sort of common code and don't duplicate? good idea

Reply:
We'll implement that in a future PR https://progress.opensuse.org/issues/109295